lib/jekyll/command.rb

```diff
@@ -0,0 +1,22 @@
+module Jekyll
+  class Command
+    class << self
+      JEKYLL_ADMIN_OVERRIDES = {
+        "watch"    => false,
+        "detached" => false,
```

Comment:
Yes, but in hindsight, it's not needed as it would detach both server processes, so we should be fine.